### DISCUSSION TITLE
qfield for qgis - unstable

### DISCUSSION THREAD

**Discussion by sbec101** [opengisch/QField#2645] [Q&A]
Hello everyone, I have been successfully been working with qfield and qfield cloud on a brand new galaxy active tab 3 but when I try to use my 2019 galaxy tab a8 with android 10 on it , the app simply crashes almost right away. I have 2 other people trying with 1-3 year android tablets with the same results. I know this is early access/unstable but does anybody have any idea if qfield cloud will be able to work with older ( slightly) tablets? Is anyone experiencing these issues? 
thx

**Comment by nirvn** [comment]
@sbec101 , on your tab a8, did you do a clean install of 2.0 beta or updated from 1.10? If you had a previous version installed, try to remove that first.

Which beta version did you try? And when you say it crashes almost straight away, when does the crash happen exactly? Before loading a project?

**Reply by sbec101** [reply]
Hi, sorry for the late reply. My tablet has the regular qfield app installed that I use ( version 1.10.0). I then installed the qfield unstable before the march24, 2022 update and that app would open up, and within 2-3 seconds crash ( turn off) without any kind of user input. I then tried to update that app just now and the same thing happens. Now I completely removed the qfield unstable app and downloaded the latest one, same thing as above. Now I have removed all qfield apps from my tablet, rebooted device and downloaded the qfield unstable app, so now with only the qfield unstable app on my device the app opens up and withing 1-3 seconds turns off with no warnings or messages. Please HELP ! , thx

**Comment by suricactus** [comment]
@sbec101 since a week your crash should have been gone by now by the successful efforts of @nirvn to fix your issue. Enjoy QField-ing!
